layouts/integrations/single.html

```diff
@@ -1,12 +1,18 @@
 {{ define "main" }}
+    {{$title := .Params.integration_title | default .Title}}
     <div class="row">
         <div class="col-12 order-1">
-            <h1 id="pagetitle">{{ if .Params.integration_title }}{{ .Params.integration_title }}{{ else }}{{.Title}}{{ end }}</h1>
+            <h1 id="pagetitle">{{$title}}</h1>
```

Comment:
Much nicer! i'm just thinking we don't need to assign this to $title if we aren't using it anywhere else. So maybe just ```<h1 id="pagetitle">{{.Params.integration_title | default .Title}}</h1>```

Reply:
that works 👍